app.js

```diff
@@ -1,10 +1,15 @@
 const express = require("express")
 const {getAllCategories} = require("./controllers/categories.controller.js")
+const {getReviews} = require("./controllers/reviews.controller.js")
 const {errorHandler} = require("./controllers/error.controller.js")
 
+
 const app = express();
 
+app.use(express.json())
```

Comment:
We only need this once we start to deal with PATCH/POST requests, so this should be deleted for now.